COLLABORATOR_GUIDE.md

```diff
@@ -415,9 +416,9 @@ Node.js uses three Deprecation levels:
   notice indicating the deprecated status is added to the API documentation
   but no functional changes are implemented in the code. There will be no
   runtime deprecation warnings emitted for such deprecations by default.
-  Documentation-only deprecations may trigger a runtime warning when launched
-  with [`--pending-deprecation`][] flag (or its alternative,
-  `NODE_PENDING_DEPRECATION=1` environment variable).
+  Documentation-only deprecations may trigger an opt-in only runtime warning
+  when Node.js is launched with the [`--pending-deprecation`][] flag (or its
+  alternative, `NODE_PENDING_DEPRECATION=1` environment variable).
```

Comment:
`(or its alternative, `NODE_PENDING_DEPRECATION=1` environment variable)` -> `or if the `NODE_PENDING_DEPRECATION` environment variable is set`.

Reply:
the value specifically needs to be `NODE_PENDING_DEPRECATION=1`